packages/sdk-ui-ts/src/utils/helpers.ts

```diff
@@ -31,3 +32,17 @@ export const getTokenLogoFromTokenType = (tokenType: TokenType) => {
       return getTokenLogoWithVendorPathPrefix('injective-black-fill.svg')
   }
 }
+
+export const getValidatorLogoWithVendorPathPrefix = (
```

Comment:
Let's add a simple test coverage for this, we can use [this](https://github.com/InjectiveLabs/injective-ts/blob/master/packages/sdk-ts/src/utils/address.spec.ts) as an reference.

Reply:
ahh yes! meant to specify that the draft's in PR because I still need to add tests. and thanks for the reference man. will check it out

Reply:
added a test based off the reference